app/models/authentication.rb

```diff
@@ -1,5 +1,7 @@
+include UserSessionManagement
+
 class Authentication < ApplicationRecord
-  belongs_to :user, inverse_of: :authentications
+  belongs_to :user, inverse_of: :authentications, optional: true
```

Comment:
Oddly enough, optional is correct here because the column is nullable in the DB.